tests/consensus_tests/test_resharding_replay.py

```
@@ -0,0 +1,88 @@
+    target_peer_id = get_cluster_info(peer_uris[target_idx])["peer_id"]
+
+    create_collection(peer_uris[0], collection=COLLECTION, shard_number=2, replication_factor=1)
+    wait_collection_exists_and_active_on_all_peers(COLLECTION, peer_uris)
+
+    raft_state = target_dir / "storage" / "raft_state.json"
+    config = target_dir / "storage" / "collections" / COLLECTION / "config.json"
+    new_shard_dir = target_dir / "storage" / "collections" / COLLECTION / "2"
+
+    commit_before = json.loads(raft_state.read_text())["state"]["hard_state"]["commit"]
+
+    resp = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)